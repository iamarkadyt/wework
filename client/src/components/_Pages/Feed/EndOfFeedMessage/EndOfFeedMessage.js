import React from 'react'
import { string } from 'prop-types'

const endOfFeedMessage = ({ endOfFeed }) => (
    <p className="Feed-message">
        {endOfFeed}
    </p>
)

endOfFeedMessage.propTypes = {
  endOfFeed: string.isRequired
}

export default endOfFeedMessage
