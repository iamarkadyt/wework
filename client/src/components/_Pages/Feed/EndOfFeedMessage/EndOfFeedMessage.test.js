import React from 'react'
import EndOfFeedMessage from './EndOfFeedMessage'
import { shallow } from 'enzyme'
import cloneDeep from 'lodash.clonedeep'

describe('EndOfFeedMessage', () => {
  let mountedComponent, props

  const getMockProps = () => {
    return cloneDeep({
      endOfFeed: 'You have reached the end of your feed'
    })
  }

  const comp = () => {
    if (!mountedComponent) {
      mountedComponent = shallow(<EndOfFeedMessage {...props} />)
    } 
    return mountedComponent
  }

  beforeEach(() => {
    mountedComponent = undefined
    props = getMockProps()
  })

  it('matches snapshot', () => {
    expect(comp()).toMatchSnapshot()
  })
})
