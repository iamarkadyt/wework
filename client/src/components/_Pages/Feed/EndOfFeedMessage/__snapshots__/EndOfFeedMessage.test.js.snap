// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EndOfFeedMessage matches snapshot 1`] = `
<p
  className="Feed-message"
>
  You have reached the end of your feed
</p>
`;
